src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/IRunnableProjectConfig.cs

```diff
@@ -54,5 +55,7 @@ public interface IRunnableProjectConfig
         IReadOnlyDictionary<string, IBaselineInfo> BaselineInfo { get; }
 
         bool HasScriptRunningPostActions { get; set; }
+
+        DateTime? ConfigTimestampUtc { get; set; }
```

Comment:
I don't think this is necessary or desired. The timestamp is a template cache only concept, it won't exist in the actual template.json.